### DISCUSSION TITLE
Not working the Transfer File nightly release

### DISCUSSION THREAD

**Discussion by mikekgr** [rustdesk/rustdesk#4230] [General]
Dear Sirs, Dear @rustdesk 
I have installed the Build Date: 2023-04-28 00:22 in my two Windows 10, Windows 11 PCs and selecting the Transfer File today (I have to check it for weeks!), it is opening the particular window and then its stay forever at "Connecting..." action. If I choose to connect to the remote PC then, the connection is done perfectly fine. Please check and let me know.
Please note that I am using private rendezvous server.

Thanks and Best Regards,
Mike Kranidis

**Comment by rustdesk** [comment]
@21pages please confirm this

**Comment by 21pages** [comment]
Yes, stuck in connecting

FingerPrintState not initialized on file transfer

**Reply by rustdesk** [reply]
https://github.com/rustdesk/rustdesk/pull/4231

**Comment by mikekgr** [comment]
I confirm that is working now in the latest nightly release Build Date: 2023-04-29 03:16
Thanks a lot guys !
